### DISCUSSION TITLE
Paladinos com problemas nos Pontos de Mana

### DISCUSSION THREAD

**Discussion by PaladinodeValkaria** [YuriAlessandro/gerador-ficha-tormenta20#310] [Problemas]
Segundo diz o poder Abençoado "Abençoado: **Você soma seu bônus de Carisma** no seu total de pontos de mana no 1º nível. Além disso, torna-se devoto de uma divindade disponível para paladinos (Azgher, Khalmyr, Lena, Lin-Wu, Marah, Tanna-Toh, Thyatis, Valkaria)." Infelizmente, nenhum paladino está recebendo os PM extras do Carisma.

Testando o Arcanista - Caminho do Feiticeiro, os PM extras do Carisma estão sendo contabilizados normalmente.

**Comment by PaladinodeValkaria** [comment]
Bom saber que o meu comentário ficou mais de 20 DIAS no vácuo

**Comment by YuriAlessandro** [comment]
Desculpa amigo, não tinha visto sua mensagem e não estou com tanto tempo para dar uma olhada no projeto ultimamente. Vou deixar registrado aqui, e vou corrigir na próxima atualização. Obrigado pelo feedback.

**Comment by YuriAlessandro** [comment]
Problema corrigido na versão 2.2

**Reply by PaladinodeValkaria** [reply]
Muito obrigado. Já testei e está funcionando bem.
